thirdeye/thirdeye-frontend/app/styles/pods/rca.scss

```diff
@@ -147,11 +147,15 @@
   }
 }
 
-.color-box {
+.container-fluid {
+  padding: 0px;
+}
+
+.color-circle {
```

Comment:
fyi, we've built a `.entity-indicator` class, it currently lives in `filter-bar.scss` but we will soon move it into its own file.